src/ContractInteract/OSTPrime.js

```diff
@@ -43,6 +60,121 @@ class OSTPrime {
     this._wrapRawTx = this._wrapRawTx.bind(this);
     this.unwrap = this.unwrap.bind(this);
     this._unwrapRawTx = this._unwrapRawTx.bind(this);
+    this.setCoGateway = this.setCoGateway.bind(this);
+    this.setCoGatewayRawTx = this.setCoGatewayRawTx.bind(this);
+    this.initialize = this.initialize.bind(this);
+    this.initializeRawTx = this.initializeRawTx.bind(this);
+  }
+
+  /**
+   * Setup for OSTPrime contract. Deploys the contract and initializes
+   * it. See {@link OSTPrime#initialize}.
+   *
+   * @param {Web3} web3 Web3 object.
+   * @param {OSTPrimeSetupConfig} config OSTPrime setup configuration.
+   * @param {Object} txOptions Transaction options.
+   *
+   * @param {Promise<OSTPrime>} Promise containing the OSTPrime instance that
+   *                           has been set up.
+   */
+  static setup(web3, config, txOptions) {
+    if (!config.valueToken) {
+      throw new Error(
+        'Mandatory configuration "valueToken" missing. Provide EIP20 token contract address.',
+      );
+    }
+
+    if (!config.organization) {
+      throw new Error(
+        'Mandatory configuration "organization" missing. Set config.organization address.',
+      );
+    }
+
+    OSTPrime.validateSetupConfig(config);
+
+    const deployParams = Object.assign({}, txOptions);
+    deployParams.from = config.deployer;
+    deployParams.gasPrice = 0;
+
+    // 1. Deploy the Contract
+    const ostPrime = OSTPrime.deploy(
+      web3,
+      config.valueToken,
+      config.organization,
+      deployParams,
+    );
+
+    // 2. Initialize.
+    const initializedOstPrime = ostPrime.then((contract) => {
+      const valueToTransfer = Web3.utils.toWei('800000000');
+      const ownerParams = Object.assign({}, deployParams);
+      ownerParams.from = config.chainOwner;
```

Comment:
What's a `chainOwner`?

Reply:
`chainOwner` is AFAIK the creator of an auxiliary chain who is supposed to hold all base tokens of the auxiliary chain and then transfers them to the OSTPrime contract during initialization. Very OST-specific.